zeppelin-server/src/main/java/org/apache/zeppelin/utils/RequestUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.utils;
+
+import java.util.*;
+
+/**
+ * Tools for correct request
+ */
+public class RequestUtils {
+  
+  public static void removeEmptyUser(HashSet set) {
+    for (Iterator iter = set.iterator(); iter.hasNext();) {
+      Object item = iter.next();
+      if (item == null || ((String) item).trim().isEmpty()) {
+        iter.remove();
```

Comment:
If you assume item as String, how about assuming the first parameter as `HashSet<String>`? It's more reasonable. And it is definitely kind of utils but this is used by `NotebookRestApi` for now, I suggest this method is located `NotebookRestApi` class. How about it?